src/app/application.h

```diff
@@ -103,6 +106,10 @@ private slots:
     QPointer<WebUI> m_webui;
 #endif
 
+    // FileLog
+    bool m_fileLogEnabled;
+    QPointer<FileLogger> m_fileLogger;
```

Comment:
I see. Thank you for the reply.

On 13 March 2016 at 16:38, sledgehammer999 <notifications@github.com> wrote:

> In src/app/application.h
> <https://github.com/qbittorrent/qBittorrent/pull/4681#discussion_r55938801>
> :
>
> > @@ -103,6 +106,10 @@ private slots:
> >      QPointer<WebUI> m_webui;
> >  #endif
> >
> > +    // FileLog
> > +    bool m_fileLogEnabled;
> > +    QPointer<FileLogger> m_fileLogger;
>
> We do cleanup in Application::cleanup() instead of destructor. The reason
> is we need to control the shutdown process on Windows otherwise windows
> kills us aka unclean exit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qbittorrent/qBittorrent/pull/4681/files#r55938801>.
>